### ISSUE TITLE
A11y - Cannot create Date & Time Options 

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5665]
The users are not able to add date or time options at the moment

*How to reproduce*

# Create a new product template 
# Go to {{Options}} card > Click to {{Add Option}} and try to add Date/Time field

*Expected behaviour:* Be able to adjust Date/Time modal and save it

*Actual behaviour*: There is an error in console and the page is frozen

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-755**



[created from CUS-755, comment 516064]
